Makefile

```diff
@@ -413,6 +413,9 @@ PILOT_TEST_BINS:=${ISTIO_OUT}/pilot-test-server ${ISTIO_OUT}/pilot-test-client $
 $(PILOT_TEST_BINS):
 	CGO_ENABLED=0 go build ${GOSTATIC} -o $@ istio.io/istio/$(subst -,/,$(@F))
 
+hyperistio:
```

Comment:
Interesting ... never heard of "hyper" istio.  Sounds like it hasn't been taking it's meds :)

Reply:
It's a play on hyperkube - which is used in minikube - basically all k8s servers plus etcd linked in one binary.